tests/IceRpc.Compressor.Tests/AssemblyInfo.cs

```diff
@@ -1,3 +0,0 @@
-// Copyright (c) ZeroC, Inc.
-
-[assembly: NUnit.Framework.Timeout(8000)]
```

Comment:
When I run "dotnet test", I want a test that takes more than 8s to fail. This shouldn't be a CI-only behavior.

This also seems unrelated to this dependency upgrade.

Reply:
See also https://docs.nunit.org/articles/nunit/writing-tests/attributes/cancelafter.html

Timeout is deprecated, and the new CancelAfter requires to upgrade all test to deal with an extra cancelation token.

When not using CI we can just attach a debugger to the hanging test